source/getting-started/index.md

```diff
@@ -45,18 +56,23 @@ download and run [this Git installer](http://git-scm.com/download/win).
 
 ### Watchman (optional)
 
-On Mac and Linux, you can improve file watching performance by installing [Watchman](https://facebook.github.io/watchman/docs/install.html).
+On Mac and Linux, you can improve file watching performance and receive alerts when files change or get
+deleted/added by installing [Watchman](https://facebook.github.io/watchman/docs/install.html).
 
 ## Installation
 
-Install Ember using npm:
+Ember and Ember CLI are installed using npm.
```

Comment:
might be misleading, since the last Ember deployed to npm is 1.0.0-beta.2 or something like that?